R/init.R

```diff
@@ -39,7 +44,8 @@ vt_use_validation <- function(pkg = ".") {
 #'
 #' @export
 #' @importFrom usethis create_package
-vt_create_package <- function(pkg = ".", ...) {
+vt_create_package <- function(pkg = ".", ..., fields = list(), rstudio = rstudioapi::isAvailable(),
+                              roxygen = TRUE, check_name = TRUE, open = rlang::is_interactive()) {
```

Comment:
Is the idea to pass these arguments into `create_package`?

Reply:
yes. 🤦 will fix in a moment